README.md

```diff
@@ -47,7 +47,7 @@ your own repository.
 4. `cp .env.example .env`
 5. `cp .env.development.example .env.development`
 6. `cp .env.test.example .env.test`
-7. `mv README.example.md README.md`
+7. `curl -o README.md https://raw.githubusercontent.com/abtion/muffi/master/README.example.md`
```

Comment:
I feel like I'm missing out on something here. If this gets merged, won't the curl command then return a 404?

Reply:
Jesper is right :O
is the URL pointing to the wrong repo?